src/core/HeaderCell/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { forwardRef } from "react";
+import Icon, { IconNameToSizes } from "../Icon";
+import Tooltip, { TooltipProps } from "../Tooltip";
+
+import {
+  HeaderCellExtraProps,
+  StyledSortingIcon,
+  StyledTableHeader,
+} from "./style";
+
+interface HeaderCellContentProps {
+  active?: boolean;
+  textPosition?: "left" | "right";
```

Comment:
should we change this to include a 'center' option for consistency, if we are doing it for other cells?

Reply:
Thanks for spotting this one, I added it in the final push 👍🏻